commands/webhook.go

```diff
@@ -155,6 +212,19 @@ func createIncomingWebhookCmdF(c client.Client, command *cobra.Command, args []s
 		return errors.New("Unable to find user '" + userArg + "'")
 	}
 
+	var owner *model.User
+	ownerArg, _ := command.Flags().GetString("owner")
+	if viper.GetBool("local") && ownerArg == "" {
+		return errors.New("owner flag should be set when this command is run in local mode")
+	}
+
+	if ownerArg != "" {
+		owner = getUserFromUserArg(c, ownerArg)
+		if owner == nil {
+			return errors.New("unable to find owner user: " + ownerArg)
```

Comment:
```suggestion
			return errors.Errorf("unable to find owner user: ", ownerArg)
```

Reply:
Addressed in 49bf106

Reply:
https://media.giphy.com/media/4xpB3eE00FfBm/giphy.gif